### PR TITLE
Cordova iOS beginners guide

### PR DESCRIPTION
**Pull request by T1TAN1UM**
guide for developing hybrid iOS cordova app

### CODE REVIEW

**Comment by T1TAN1UM** [comment]
@texas2010 what do i need to do next ?

**Comment by T1TAN1UM** [comment]
@texas2010 @Bouncey please suggest further steps

**Comment by MrAsrani1312** [comment]
@T1TAN1UM Do you have more content to be added ? I'll advise you to shorten the length of an article if possible . Approved for merge. Thanks for your awesome contribution 😄

**Comment by T1TAN1UM** [comment]
@MrAsrani1312  @texas2010 @Bouncey no more content to be added, to maintain completeness i think i cannot reduce content.

**Comment by T1TAN1UM** [comment]
sorry closed by mistake

File: src/pages/hybrid-application-development/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (Bouncey) [inline src/pages/hybrid-application-development/index.md]: Why are we adding this page again?

This is the same as https://guide.freecodecamp.org/mobile-app-development

File: src/pages/hybrid-application-development/fcm-integration-for-cordova-hybrid-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
+---
```
Comment (Bouncey) [inline src/pages/hybrid-application-development/fcm-integration-for-cordova-hybrid-apps/index.md]: This page already exists at https://guide.freecodecamp.org/mobile-app-development/firebase-cloud-messaging-integration-for-cordova-hybrid-apps

**Comment by T1TAN1UM** [comment]
@Bouncey @MrAsrani1312 @systimotic updated the fork to use the latest *mobile-app-development* category as upstream

**Pull request closed without merging**